services/app-api/src/testHelpers/emailerHelpers.ts

```diff
@@ -439,10 +489,27 @@ const mockContractAmendmentFormData = (
             },
         ],
         actuaryContacts: [],
+        rateProgramIDs: ['3fd36500-bf2c-47bc-80e8-e7aa417184c5'],
         ...submissionPartial,
     }
 }
 
+const findProgramsHelper = (
+    stateCode: string,
+    programIDs: string[]
+): ProgramType[] | Error => {
+    const programs = mockMNState().programs.filter((program) =>
```

Comment:
docs: please add a comment about why we put state specific logic in this test helper

Reply:
This function was removed entirely since we moved `findPrograms` to emailer functions, it was easier to use actual programs for each state. For instance testing `CHIP` submission, I edited the submission to be `MS` state and programs for `MS`. We could create a more robust mock of a state, but I feel this PR has gotten pretty long.